badgerdb/badgerdb.go

```diff
@@ -127,7 +167,14 @@ func NewStore(options Options) (Store, error) {
 
 	// Open the Badger database located in the options.Dir directory.
 	// It will be created if it doesn't exist.
-	opts := badger.DefaultOptions(options.Dir)
+	opts := badger.DefaultOptions(options.Dir).
```

Comment:
Should we add some tests on our own to test some of the new features? Or do we trust BadgerDB tests because we're not doing anything on our own here, it's all just BadgerDB features?